File: Python Exams/Basic Exam Collection/Basics Online Exam - 15 and 16 June 2019/01. Movie Profit/01. Movie Profit.py
move_name = input()
days = int(input())
tickets_count = int(input())
ticket_price = float(input())
cinema_percent = int(input())

cinema_price = tickets_count * ticket_price  * days
subtotal = cinema_price * cinema_percent / 100
total = cinema_price - subtotal
print(f'The profit from the movie {move_name} is {total:.2f} lv.')


'''
The Jungle
22
20500
9.37
30
'''

File: JS Exams/01.  Basics Exam Collection/Basics Online Exam - 15 and 16 June 2019/01. Movie Profit/01. Movie Profit.js
function movieProfit(moveName, days, ticketsCount, ticketPrice, cinemaPercent){
    let cinemaPrice = ticketsCount * ticketPrice  * days
    let subtotal = cinemaPrice * cinemaPercent / 100
    let total = cinemaPrice - subtotal

    return `The profit from the movie ${moveName} is ${total.toFixed(2)} lv.`
}

//console.log(movieProfit('The Programmer', 20, 500, 7.50, 7))
